lstchain/calib/camera/calibration_calculator.py

```diff
@@ -241,6 +241,7 @@ def calculate_calibration_coefficients(self, event):
         if self.use_scaled_low_gain:
             calib_data.unusable_pixels[constants.LOW_GAIN] = calib_data.unusable_pixels[constants.HIGH_GAIN]
             calib_data.dc_to_pe[constants.LOW_GAIN] = calib_data.dc_to_pe[constants.HIGH_GAIN] * self.hg_lg_ratio
+            calib_data.time_correction[constants.LOW_GAIN] = calib_data.time_correction[constants.HIGH_GAIN]
```

Comment:
Discussed offline: I asked whether we can use HG for the time correction of LG given that the two use slightly different signal paths and different readout chips, answer is yes:
![comparison_timecorr](https://github.com/cta-observatory/cta-lstchain/assets/5488440/7d6ced7b-742b-4286-824b-c47fba6d8c8d)